app/models/customer.py

```diff
@@ -0,0 +1,23 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
+    
+
+    def customer_to_json(self):
+        return {
+            "id": self.task_id,
```

Comment:
This looks like it was copied from Tasklist and not changed.

```suggestion
            "id": self.customer_id,
```